demos/dqx_demo_asset_bundle/dqx_demo_notebook.py

```diff
@@ -0,0 +1,630 @@
+# Databricks notebook source
```

Comment:
Would it be possible to reuse the existing manufacturing demo? it's largely copy and past. Just a consideration